assets/livechat-logo.svg

```diff
@@ -0,0 +1 @@
4-4.28 11 11 0 0 0-3.75-2.84 11.59 11.59 0 0 0-5-1A13.07 13.07 0 0 0 82 39a12 12 0 0 0-4.1 3.2 13.89 13.89 0 0 0-2.49 4.75 19.79 19.79 0 0 0-.84 5.85 20.06 20.06 0 0 0 .81 5.84 12.54 12.54 0 0 0 2.52 4.62 11.57 11.57 0 0 0 4.41 3 17.32 17.32 0 0 0 6.49 1.09 24.74 24.74 0 0 0 9.2-1.66 14.26 14.26 0 0 0-.47-3.28 13.3 13.3 0 0 0-1-2.62 20.6 20.6 0 0 1-7.19 1.37 8.58 8.58 0 0 1-5.25-1.48 6 6 0 0 1-2.21-4.75h17.47a20.76 20.76 0 0 0 .26-3.61 17.75 17.75 0 0 0-.78-5.38zM82 49.52a7.38 7.38 0 0 1 1.63-4.31 5.07 5.07 0 0 1 4-1.53 4.36 4.36 0 0 1 3.6 1.61 7.08 7.08 0 0 1 1.39 4.23z" fill="#3a3a3a" fill-rule="evenodd" /><path d="M127.28 59.25a22.55 22.55 0 0 1-3.31.92 18.06 18.06 0 0 1-3.41.28q-10.86 0-10.87-13.33 0-6.72 2.71-10.13a9.28 9.28 0 0 1 7.74-3.42 17.22 17.22 0 0 1 3.36.28 23.14 23.14 0 0 1 3.26.92c.42-1.05.8-2.1 1.15-3.14a12.36 12.36 0 0 0 .63-3.19 19.9 19.9 0 0 0-4.54-1.37 30.45 30.45 0 0 0-4.65-.33 18 18 0 0 0-7.59 1.53 15.79 15.79 0 0 0-5.61 4.26 19 19 0 0 0-3.54 6.47 26.11 26.11 0 0 0-1.21 8.11 27.07 27.07 0 0 0 1.15 8.08 18.18 18.18 0 0 0 3.45 6.43 15.69 15.69 0 0 0 5.64 4.21 19.06 19.06 0 0 0 7.83 1.5 31.52 31.52 0 0 0 4.7-.33 24.55 24.55 0 0 0 4.91-1.36 22.55 22.55 0 0 0-.66-3.25 21.76 21.76 0 0 0-1.14-3.14zm19.16-21.47a8.81 8.81 0 0 0-2.88.43 9.07 9.07 0 0 0-2.16 1.07 8.87 8.87 0 0 0-1.49 1.31c-.41.46-.73.85-1 1.18v-15a14.77 14.77 0 0 0-1.89-.21c-.62 0-1.24-.06-1.83-.06s-1.21 0-1.84.06a13.81 13.81 0 0 0-1.84.21v40a17.3 17.3 0 0 0 1.87.22h3.64a16.81 16.81 0 0 0 1.89-.22v-14.9a11.16 11.16 0 0 1 .51-3.71 6.13 6.13 0 0 1 1.25-2.22 3.89 3.89 0 0 1 1.71-1.06 6.66 6.66 0 0 1 1.84-.27 3.19 3.19 0 0 1 3 1.58 9.32 9.32 0 0 1 .89 4.53v16a17.55 17.55 0 0 0 1.9.22h3.67a17.37 17.37 0 0 0 1.89-.22V48.65q0-5.62-2.34-8.25c-1.49-1.75-3.78-2.62-6.79-2.62zm22 0a27.06 27.06 0 0 0-4.31.38 28 28 0 0 0-4.61 1.2 11 11 0 0 0 .36 3A15.53 15.53 0 0 0 161 45a18 18 0 0 1 6-1.14 15.87 15.87 0 0 1 2.39.16 4.66 4.66 0 0 1 1.81.63 3.27 3.27 0 0 1 1.18 1.35 4.69 4.69 0 0 1 .42 2.13v.66a38.13 38.13 0 0 0-3.36-.17 19.38 19.38 0 0 0-4.2.47 12.09 12.09 0 0 0-3.81 1.53 8.49 8.49 0 0 0-2.78 2.84 8.32 8.32 0 0 0-1.08 4.4 10.12 10.12 0 0 0 .87 4.42 7.62 7.62 0 0 0 2.47 3 10.34 10.34 0 0 0 3.83 1.64 22.69 22.69 0 0 0 4.91.49 28.84 28.84 0 0 0 5.93-.55 41.3 41.3 0 0 0 4.42-1.22V49.25q0-6-3.07-8.74t-8.49-2.73zm4.36 23.16l-1.37.24a8.33 8.33 0 0 1-1.57.14c-.46 0-1 0-1.6-.08a5.64 5.64 0 0 1-1.74-.44 3.46 3.46 0 0 1-1.39-1.12 3.58 3.58 0 0 1-.58-2.18 3.39 3.39 0 0 1 1.5-3.09 7.47 7.47 0 0 1 4.07-1 11 11 0 0 1 1.31.09c.49.05.95.12 1.37.19zM199 62.6a11.06 11.06 0 0 0-.52-2.15l-1.34.24a8.85 8.85 0 0 1-1.55.14 6 6 0 0 1-1.73-.22 2.51 2.51 0 0 1-1.23-.82 4.11 4.11 0 0 1-.77-1.64 11.77 11.77 0 0 1-.26-2.73V44.33h6.35a13 13 0 0 0 .21-1.58c0-.47.06-1 .06-1.48s0-.92-.06-1.36a11.65 11.65 0 0 0-.21-1.48h-6.35v-8.24h-1.21l-9.81 13.05.26 1.09h3.41v12.78q0 5.25 2.52 7.57t7.3 2.32a20.66 20.66 0 0 0 5-.6c0-.25.06-.52.07-.79s0-.61 0-1a13.92 13.92 0 0 0-.14-2.01z" fill="#d46d39" fill-rule="evenodd"/></svg>
```

Comment:
I'm not sure that we should upload LC logo here.

Reply:
I changed it to a better image
![sylvester](https://user-images.githubusercontent.com/46320952/61881289-a9795600-aef6-11e9-8f76-c5a67a24e851.jpg)

Reply:
that's the spirit! approve.